en/middleware/uxrce_dds.md

```diff
@@ -438,7 +439,96 @@ Each (`topic`,`type`) pairs defines:
 4. The message type (`VehicleOdometry`, `VehicleStatus`, `OffboardControlMode`, etc.) and the ROS 2 package (`px4_msgs`) that is expected to provide the message definition.
 
 You can arbitrarily change the configuration.
-For example, you could use different default namespaces or use a custom package to store the message definitions. 
+For example, you could use different default namespaces or use a custom package to store the message definitions.
```

Comment:
@beniaminopozzan Thanks for this - quite a lot to take in!

I have subedited. The main change is to add some structure/headings. It will now need your review.